opendds_cmake_module/cmake/Modules/FindOpenDDS.cmake

```diff
@@ -0,0 +1,71 @@
+# Copyright 2016-2018 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################################################
+#
+# CMake module for finding OpenDDS.
```

Comment:
Try removing this and using the CMake module provided by OpenDDS

Reply:
I will look at the CMake files of rmw_opendds_cpp. It seems similar to this package regarding the part that links to OpenDDS.

Reply:
See https://github.com/oci-labs/rosidl_typesupport_opendds/pull/3 PR #3 of this repo